### PR TITLE
Avoid usage of ipython_genutils

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

**Comment by blink1073** [comment]
Once this is merged all of the `jupyter_server` projects will be off of `ipython_genutils`.  I'm waiting for #723 and getting all green before merging.

**Comment by blink1073** [comment]
Actually, let's hold off on this one until there is a JupyterLab 3.3.x release with https://github.com/jupyterlab/jupyterlab/pull/12202, since this might break JupyterLab users.

**Review by minrk — APPROVED** [review]

**Pull request merged**
